devutils/git-hooks/pre-commit

```diff
@@ -44,33 +51,61 @@ PreCommit::BuildChecks () {
         go install github.com/bazelbuild/buildifier/buildifier
     fi
 
-    output=$("$GOPATH/bin/buildifier" -mode=check $filepath)
+    output=$("$GOPATH/bin/buildifier" -mode=check "$filepath")
     if [ -n "$output" ]; then
         Log::Warning "$output"
-        Log::Failure "formatting error in a BUILD file. Aborting..."
+        Log::Error "formatting error in a BUILD file."
+    fi
+}
+
+
+# Rules checking file is correctly registered in the commit
+PreCommit::GitChecks () {
+    local filepath="$1"
+
+    # Ensure commit contains the same file that the one currently in the project
+    if [ "$(git status --porcelain "$filepath" | cut -c2)" != ' ' ]; then
```

Comment:
The use of `if` statement is not consistent with the `[` tests used previously. To avoid stacking all the code in a single line, you can use 
```
[ myCondition ] && \
    myErrorLog
```